data/sheragi/the emerald sword.txt

```diff
@@ -0,0 +1,854 @@
+		log "People" "Desyo" `A brilliant Hai xenoprogramming expert, Desyo specializes in Sheragi code and software. She's more interested in technology and electronics than people, and often coordinates the work in the Hai archeologist team despite her introversion, and lack of hierarchy. Her dedication is the grease that keeps that machine running.`
+		log "People" "Rakei" `Rakei is an engineering expert specializing in archaeological artifacts, but also very interested in new and unusual technology, often nerding out with Desyo over scientific papers. She's part of the Hai archaeologist team tasked to uncover the mystery of the Sheragi black box.`
+		log "People" "Eyama" `Uncaring about formalities and seriousness, you wouldn't guess Eyama is a very accomplished linguist specialized in Sheragi language. She also has an interest in culture and sociology in general, and has a lot more knowledge about human culture than you'd expect from a Hai. Her work with the Hai archaeologist team is invaluable to deciphering Sheragi mysteries and maintaining their relationship with human partners.`
+
+mission "Sheragi Archaeology: The Emerald Sword 1"
+	landing
+	name "Black Box Decoded"
+	description "Visit Albert Foster on Zug."
+	source
+		near "Zubeneschamali" 1 100
+	destination "Zug"
+	
+	to offer
+		has "event: cube decoded"
+	
+	on offer
+		dialog
+			`You're still in the middle of the post-flight check when you receive a new message from Albert Foster.`
+			`	"Hello <first>. I have some good news! We have decoded what's in the black box. You should come visit us on Zug; we have plenty to tell you!"`
+
+mission "Sheragi Archaeology: The Emerald Sword 2"
+	landing
+	name "The Emerald Sword"
+	description "Bring an additional Jump Drive to Zug."
+	source "Zug"
+	to offer
+		has "Sheragi Archaeology: The Emerald Sword 1: done"
+	
+	on offer
+		conversation
+			`You arrive on Zug, and after contacting Albert and the three Hai archaeologists and following their direction, you find them in an ice cream shop. They're still wearing the youngster clothes, but not even bothering to hide their faces with bandannas anymore. Surprisingly, nobody seems too distressed at the sight. Rakei waves for you and stands up, coming to you with ice creams in paw.`
+			`	"Hello <first>! It seems our disguise worked better than intended. The humans around here think we're wearing a costume. Apparently young humans like to dress up as aliens?" she says confusedly.`
+			`	"I had a feeling we'd fit right in with the furry crowd." Eyama responds proudly.`
+			`	"It has been an interesting cultural exchange. Human space might be a great vacation spot for us. But we're not on vacation yet. We have a mystery to solve," she continues.`
+			`	You sit down with your group on a bench in a nearby park, mostly empty, and Desyo picks up the conversation.`
+			`	"So, the files in the black box were thankfully standard for Sheragi. Not very difficult to decode, once we managed to read them."`
```

Comment:
Should `decode` and `read` be switched here?

Reply:
It's this way around because "read" is in the computational sense. read the data first, get the ones and the zeroes, then decode it with whatever standard method the Sheragi used.